CONTRIBUTING.md

```diff
@@ -372,7 +372,7 @@ The following table outlines possible test file locations and how to invoke them
 
 | Test runner        | Test location                                                                                                                                           | Runner command (working directory is kibana root)                                       |
 | -----------------  | ------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
-| Jest               | `src/**/*.test.js`<br>`src/**/*.test.ts`                                                                                                                | `node scripts/jest -t regexp [test path]`                                               |
+| Jest               | `src/**/*.test.{js,ts}`<br>`x-pack(\/legacy)?/plugins/*.test.{js,ts}`                                                                                                                | `node scripts/jest -t regexp [test path]`                                               |
```

Comment:
I think we should avoid the specificity of the `(\/legacy)?/plugins` bit, since it's already outdated based on the roots. We should probably just select all `.test.{js,ts}` files in `x-pack`